build.sh

```diff
@@ -15,11 +15,13 @@ usage="$(basename "$0") -- script for compile Go source and build docker images
 usage:
     -t,--tag             tag name for docker images (default: latest)
     -o,--organization    organization name for docker images (default: eclipse)
+    -r,--repository      docker registry (default quay.io)
```

Comment:
I don't understand why we need three params (repo, organization, tag) instead of one `--image`... But I assume it's just done the same as we have somewhere. So, it's not critical but if you agree that one param is more convenient to use - consider reworking it

Reply:
Good suggestion, but here we build two images and image names hard-coded, that's why  I use several parameters here.
Probably make sens combine repository and organization names to the one parameter.